NETPROGRAM/ArrayExercise/Exercise01.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+
+namespace NETPROGRAM.ArrayExercise
+{
+    public class Exercise01
+    {
+
+        public List<int> CorrectNumbers(int[] array)
+        {
+            var arrayOrder = (from n in array
+                             orderby n
+                             select n).ToArray();
+            var listNumbers = new List<int>();
+            var count = 1;
+            var i = 0;
+
+            do
```

Comment:
Cái này có dạng i lúc đầu = 0, tăng 1 sau mỗi lần lặp cho đến khi đạt chỉ số nào đó --> phải dùng for